owncloudApp/src/main/java/com/owncloud/android/presentation/adapters/filelist/FileListAdapter.kt

```diff
@@ -41,6 +41,7 @@ class FileListAdapter(
     private val context: Context
 ) : RecyclerView.Adapter<FileListAdapter.ViewHolder>() {
 
+    var onItemClick: ((OCFile) -> Unit)? = null
```

Comment:
I usually add the `onClick()` method as parameter of the constructor:

`class FileListAdapter(
    private val context: Context,
    private val listener: (OCFile) -> Unit
) : RecyclerView.Adapter<FileListAdapter.ViewHolder>() {`

What do you think?